### DISCUSSION TITLE
reset admin password without ssh

### DISCUSSION THREAD

**Discussion by tasiotas** [LycheeOrg/Lychee#2284] [Q&A] [answered]
Hi,

I installed Lychee via FTP. How can I reset admin password without ssh access ?

Thank you

**Comment by ildyria** [comment]
**You can't. You need to write a small script that you upload yourself and execute it.**
As this is extremely unsafe, such thing will never be implemented in Lychee.

You can try this script in `reset_admin.php` which you place in `<path_to_lychee>/public/reset_admin.php` and then access it from your web. Do be sure to delete the script once done. I have no clue if it works but it is worth a try.

```php
<?php

exec("cd ..; php artisan lychee:reset_admin", $output, $retval);
echo "Returned with status $retval and output:\n";
print_r($output);
```

**Comment by tasiotas** [comment]
yey, it worked!

Just had to use `create_user --may-administrate` instead of `reset_admin`

Thank you

**Comment by tasiotas** [comment]
this doesnt work on v5 anymore

returns:
```
Returned with status 255 and output: Array ( )
```

I don't see any changes to `create_user`, any ideas?

**Comment by ildyria** [comment]
Have you tried `lychee:update_user <username> <password>` (without the `<` and `>` obviously) ?

**Comment by tasiotas** [comment, marked as answer]
yes

```php
<?php

exec("cd ..; php artisan lychee:update_user lycheadmin4 fhdFD34GFdd@#d", $output, $retval);
echo "Returned with status $retval and output:\n";
print_r($output);
```

**Comment by tasiotas** [comment]
I found clue in error.log
Composer was complaining about PHP version. Once I updated it to >=8.2 it worked again

sorry for trouble
